synfig-studio/plugins/lottie-exporter/common/Param.py

```diff
@@ -541,6 +541,33 @@ def recur_animate(self, anim_type):
                 self.expression = ret
                 return ret, self.expression_controllers
 
+            elif self.param[0].tag == "vectorx":
+                self.subparams["vectorx"].extract_subparams()
+                vector, eff_1 = self.subparams["vectorx"].subparams["vector"].recur_animate("vector")
+                self.expression_controllers.extend(eff_1)
+                if self.dimension == 2:
+                	ret = "[{x}[0],{x}[0]]"
+                else:
+                	ret = "{x}[0]"
+                ret = ret.format(x=vector)
+
+                self.expression = ret
+                return ret, self.expression_controllers
+
+            elif self.param[0].tag == "vectory":
+                self.subparams["vectory"].extract_subparams()
+                vector, eff_1 = self.subparams["vectory"].subparams["vector"].recur_animate("vector")
+                self.expression_controllers.extend(eff_1)
+                if self.dimension == 2:
```

Comment:
I think we can write this statement as:
`ret = "[{y}[1], {y}[1]]" if self.dimension == 2 else "{y}[1]"`
This is more compact and removes the warnings on lgtm

Reply:
Changed the statement ^__^